### PR TITLE
Redesign Team Website Using Bootstrap v5.0

### PR DESCRIPTION
**Pull request by GustavoMiguelValencia**
**Note:**
This is simply a redesign of the current team website. There were no major content additions/changes. I plan on adding/updating content on the site, but figured it was important to transition to Bootstrap before making any major additional changes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,78 +1,131 @@
...
+      <header class="bg-dark py-5">
+        <div class="container px-5">
+          <div class="row gx-5 justify-content-center">
+            <div class="col-lg-6">
+              <img class="mx-auto d-block" src="images/geeksurvey_logo_hd.png" width="200px" height="200px">
+              <div class="text-center my-5">
+                <h1 class="display-5 fw-bolder text-white mb-2">The GeekSurvey Project</h1>
+                <p class="lead text-white-50 mb-4">
+                  GeekSurvey is a tool for coordinating research. To read more about GeekSurvey, see the
+                  <a href class="text-white-50"="docs/Gerosa-GeekSurvey.pdf">Capstone project proposal</a> written by the project sponsors.
```
Comment (timgiroux) [inline index.html]: should be href="<relative link>"

File: index.html [hunk 2]
```diff
@@ -1,78 +1,131 @@
...
+      The template is licensed under MIT and can be found at https://github.com/StartBootstrap/startbootstrap-modern-business/blob/master/LICENSE.
+      -->
+      <!-- Team Information Section -->
+      <section class="py-5 bg-light" id="scroll-target">
+        <div class="container px-5 my-5">
+          <div class="row gx-5 align-items-center">
+            <div class="col-lg-6"><img class="img-fluid rounded mb-5 mb-lg-0" src="images/supergeeks_team.png" alt="Image of the Four SuperGeeks team members"/></div>
+            <div class="col-lg-6">
+              <h2 class="fw-bolder">Team Information</h2>
+              <p class="lead fw-normal text-muted mb-0"><a href class="text-muted mb-0"="supergeeks.html">SuperGeeks</a> is a team of Computer Science students at Northern Arizona University. The members of SuperGeeks are Tim Giroux, Pengfei Liu, Kyle Austria, and Gustavo Valencia.</p>
```
Comment (timgiroux) [inline index.html]: Same href problem

File: index.html [hunk 3]
```diff
@@ -1,78 +1,131 @@
...
+      The template is licensed under MIT and can be found at https://github.com/StartBootstrap/startbootstrap-modern-business/blob/master/LICENSE.
+      -->
+      <!-- Project Sponsors Section -->
+      <section class="py-5">
+        <div class="container px-5 my-5">
+          <div class="row gx-5 align-items-center">
+            <div class="col-lg-6 order-first order-lg-last"><img class="img-fluid rounded mb-5 mb-lg-0" src="images/headshots/GerosaSteinmacherStitch.jpg" alt="Image of Dr. Marco Gerosa and Dr. Igor Steinmacher"/></div>
+              <div class="col-lg-6">
+                <h2 class="fw-bolder">Project Sponsors</h2>
+                <p class="lead fw-normal text-muted mb-0">The sponsors of GeekSurvey are <a href class="text-muted mb-0"="https://directory.nau.edu/person/mag696">Marco Gerosa</a> and <a href class="text-muted mb-0"="https://www.igor.pro.br/">Igor Steinmacher.</a> Dr. Gerosa and Dr. Steinmacher are distinguished Software Engineering researchers. They have found that there is a communication problem with recruiting participants for S.E. studies. This is the problem that GeekSurvey sets out to solve. </p>
```
Comment (timgiroux) [inline index.html]: Same href problem

File: process.html [hunk 4]
```diff
@@ -1,71 +1,139 @@
...
+        <div class="row gx-5 align-items-center">
+          <div>
+            <h2 class="fw-bolder">Project Description</h2>
+            <p class="lead fw-normal text-muted mb-0">
+              As it currently stands, researchers have a difficult time connecting with willing and qualified participants for software engineering research opportunities.
+              Software engineering researchers have tried both social media apps and emails to recruit potential research participants, but to little success.
+              Currently, a response rate of 5-10% is deemed acceptable. The team here at SuperGeeks believes that we can do better. We aim to develop a secure, web-based platform
+              that will effectively and efficiently manage software engineering research recruiting efforts. Our platform aims to produce valid, reliable, and relevant data for our
+              clients and other software engineering researchers. The initial concept for this project was provided by our sponsor, in the form of a
+              <a href class="text-muted mb-0"="docs/Gerosa-GeekSurvey">Capstone Project Proposal</a>.
```
Comment (timgiroux) [inline process.html]: href

File: process.html [hunk 5]
```diff
@@ -1,71 +1,139 @@
...
+    The template is licensed under MIT and can be found at https://github.com/StartBootstrap/startbootstrap-modern-business/blob/master/LICENSE.
+    -->
+    <!-- Documents Section -->
+    <section class="py-5">
+      <div class="container px-5 my-5">
+        <div class="row gx-5 align-items-center">
+          <div>
+            <h2 class="fw-bolder">Documents</h2>
+            <p class="lead fw-normal text-muted mb-0">
+              <p><a href class="text-muted mb-0"="docs/TeamInventory.pdf">Team Inventory</a> - Information about the team members.</p>
```
Comment (timgiroux) [inline process.html]: href problem lines 113-119

File: tech.html [hunk 6]
```diff
@@ -1,53 +1,119 @@
...
+    Elements of the following code are derived from the Start Bootstrap - Modern Business v.5.0.5 Template.
+    The template is licensed under MIT and can be found at https://github.com/StartBootstrap/startbootstrap-modern-business/blob/master/LICENSE.
+    -->
+    <!-- Requirements Acquisition Section -->
+    <section class="py-5 bg-light" id="scroll-target">
+      <div class="container px-5 my-5">
+        <div class="row gx-5 align-items-center">
+          <div>
+            <h2 class="fw-bolder">Codebase</h2>
+            <p class="lead fw-normal text-muted mb-0">All code for the GeekSurvey project can be found at the <a href class="text-muted mb-0"="https://github.com/NAU-SuperGeeks">NAU-SuperGeeks</a> GitHub organization.</p>
```
Comment (timgiroux) [inline tech.html]: href

File: tech.html [hunk 7]
```diff
@@ -1,53 +1,119 @@
...
+    Elements of the following code are derived from the Start Bootstrap - Modern Business v.5.0.5 Template.
+    The template is licensed under MIT and can be found at https://github.com/StartBootstrap/startbootstrap-modern-business/blob/master/LICENSE.
+    -->
+    <!-- High Level Requirements Section -->
+    <section class="py-5">
+      <div class="container px-5 my-5">
+        <div class="row gx-5 align-items-center">
+          <div>
+            <h2 class="fw-bolder">Google Forms Demo</h2>
+            <p class="lead fw-normal text-muted mb-0"><a href class="text-muted mb-0"="docs/GoogleForms_Demo.mp4"> Click Here </a> </p>
```
Comment (timgiroux) [inline tech.html]: href

**Review by timgiroux — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (GustavoMiguelValencia) [reply index.html]: Good catch! Will fix in a quick commit shortly

**Review by timgiroux — APPROVED** [review]
LGTM

**Pull request merged**